packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -180,7 +182,11 @@ Products and features are proper nouns and should be capitalized to differentiat
 ## Grammar and formatting
 
 #### Abbreviations
-Coming in the next release
+**Do** use common abbreviations if space is limited, such as in form headings.
+- *Ref.* in place of *Reference*
+- *No.* or *#* in place of *number*
+
+*Avoid* using abbreviations in body copy. It might look "neater" but it can increase cognitive load and reduce comprehension.
```

Comment:
NIT: perhaps instead of "It might look 'neater'" we could say something like "It might reduce word count"?